deepmd/pt/utils/stat.py

```
@@ -476,26 +492,44 @@
-                merged_natoms[kk],
-                assigned_bias=assigned_atom_ener[kk],
-                rcond=rcond,
-            )
         else:
             # this key does not have global labels, skip it.
             continue
+    if "property" in atomic_output.var_defs:
+        concat_bias = []
+        concat_std = []
+        for ii in atomic_output.var_defs["property"].sub_var_name:
+            assert ii in bias_atom_e.keys()
```

Ruff (0.8.2)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)